commands/Fun/say.js

```diff
@@ -0,0 +1,19 @@
+const { Command } = require("klasa");
```

Comment:
Please add the copyright header above this line. You can find it in any piece from this repo.

Reply:
Added it